### PR TITLE
Add EAST node

### PR DESCRIPTION
**Pull request by ghost**
From http://easthub.io

### CODE REVIEW

**Comment by akx20000a** [comment]
Calling @gamalielhere 😄

File: app/scripts/nodes.js [hunk 1]
```diff
@@ -270,6 +271,19 @@ nodes.nodeList = {
...
         'service': 'egem.io',
         'lib': new nodes.customNode('https://jsonrpc.egem.io/custom', '')
+    },
+    'east': {
+        'name': 'EAST',
+        'blockExplorerTX': 'https://explorer.easthub.io/tx/[[txHash]]',
+        'blockExplorerAddr': 'https://explorer.easthub.io/addr/[[address]]',
+        'type': nodes.nodeTypes.EAST,
+        'eip155': true,
+        'chainId': 9,
```
Comment (gamalielhere) [inline app/scripts/nodes.js]: Is this accurate? Seems like chainId 9 is registered to Counterparty.

Comment (ghost) [reply app/scripts/nodes.js]: @gamalielhere Yup. Checked the wrong number sorry 😄

File: app/scripts/nodes.js [hunk 2]
```diff
@@ -270,6 +271,19 @@ nodes.nodeList = {
...
         'service': 'egem.io',
         'lib': new nodes.customNode('https://jsonrpc.egem.io/custom', '')
+    },
+    'east': {
+        'name': 'EAST',
+        'blockExplorerTX': 'https://explorer.easthub.io/tx/[[txHash]]',
+        'blockExplorerAddr': 'https://explorer.easthub.io/addr/[[address]]',
+        'type': nodes.nodeTypes.EAST,
+        'eip155': true,
+        'chainId': 7,
```
Comment (gamalielhere) [inline app/scripts/nodes.js:307]: 7 node is also taken. Please refer to: https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Comment (ghost) [reply app/scripts/nodes.js:307]: @gamalielhere Should chainid similar with slip-44 number? We already registered in 7 with parity 😂

**Review by gamalielhere — APPROVED** [review]

**Comment by gamalielhere** [comment]
@akx20000a Can you resolve the conflict? I'll get this merged afterwards

**Comment by akx20000a** [comment]
@gamalielhere Sure 😄

**Comment by ghost** [comment]
@gamalielhere Done Rebase 😄

**Pull request merged**